docs/users/navigation/joysticks.md

```diff
@@ -0,0 +1,129 @@
+| Left Joystick left/right | Orbit around focus left/right |
+| Left Joystick Press | NA |
+| Right Joystick up/down | Pan camera up/down |
+| Right Joystick left/right | Pan camera left/right <br/>If LB or RB is pressed: Roll camera |
+| Right Joystick Press | NA |
+| Select | NA |
+| Start | NA |
+
+The LB and RB buttons switches the roll mode to local or global respectivly. The difference between these two is that the local roll mode rolls the camera around the center of the screen, while the global roll mode rolls the camera around the current focus.
+
+## PS4
+The image below shows a map over the buttons and joysticks on a PS4 controller.
+
+![](images/ps4.png)
+
+The navigation using a PS4 controller in OpenSpace is defined in the PS4 asset file <code>(ps4.asset)</code>. The table below will give an overview of what each button or joystick on the PS4 controller does in OpenSpace. NA in the table specifies that this button or joystick has no functionality in the default version of OpenSpace.
+
+| Button or joystick | Description |
+|--------------------|-------------|
+| Cross | Toggle zoom friction |
+| Circle | Toggle rotation friction |
+| Square | Switch focus to Earth |
+| Triangle | Switch focus to Mars |
+| L1 | Pressed: Switch to local roll mode <br/>Released: Switch back to normal mode |
+| L2 | Zoom out |
+| R1 | Pressed: Switch to global roll mode <br/>Released: Switch back to normal mode |
+| R2 | Zoom in |
+| Up | NA |
+| Right | NA |
+| Left | Toggle roll friction |
+| Down | NA |
+| Left Joystick up/down | Orbit around focus up/down |
+| Left Joystick left/right | Orbit around focus left/right |
+| Left Joystick Press | NA |
+| Right Joystick up/down | Pan camera up/down |
+| Right Joystick left/right | Pan camera left/right <br/>If L1 or R1 is pressed: Roll camera |
+| Right Joystick Press | NA |
+| Share | NA |
+| Options | NA |
+| Touch Pad | NA |
+| PS | NA |
+
+The L1 and R1 buttons switches the roll mode to local or global respectivly. The difference between these two is that the local roll mode rolls the camera around the center of the screen, while the global roll mode rolls the camera around the current focus.
+
+## SpaceMouse
+The SpaceMouse is a joystick with 6 degrees of freedom that is sold by the company [3Dconnexion](https://3dconnexion.com/uk/spacemouse/). There are a few different versions of it and therefore there are a few different versions of the asset files that specify the navigation. The versions that are currently supported (as of release 0.18.0) is the SpaceMouse Compact <code>(space-mouse-compact.asset)</code> and the SpaceMouse Enterprise <code>(space-mouse-enterprise.asset)</code>, both of which can be in wireless mode <code>(space-mouse-compact-wireless.asset and space-mouse-enterprise-wireless.asset</code>). The image below is a map of the different movements of the SpaceMouse and a translation from the terminology used by 3Dconnexion (3D) and the terminology used by OpenSpace (OS).
```

Comment:
The SpaceMouse is a joystick with 6 degrees of freedom that is sold by the company [3Dconnexion](https://3dconnexion.com/uk/spacemouse/). There are a few different versions of it and therefore there are a few different versions of the asset files that specify the navigation. The versions that are currently supported (as of release 0.18.0) is the SpaceMouse Compact (<code>space-mouse-compact.asset</code>) and the SpaceMouse Enterprise (<code>space-mouse-enterprise.asset</code>). Both of these can also be in wireless mode, for which specific asset files : <code>space-mouse-compact-wireless.asset</code> and <code>space-mouse-enterprise-wireless.asset</code>. The image below is a map of the different movements of the SpaceMouse and a translation of the terminology used by 3Dconnexion (3D) and the terminology used by OpenSpace (OS).

Reply:
Or actually, maybe make a table for the space mouse types and their asset file?